### PR TITLE
Add RUNBOOK.md

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+## Dependent Products
+
+* ftcom
+* fdi-markets
+
+## Dependencies
+
+* origami-navigation-service-data
+* dyn-dns
```
Comment (JakeChampion) [inline RUNBOOK.md]: Should this also be on all the other services runbooks?

File: RUNBOOK.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+* ft-fastly
+
+## Healthchecks
+
+* origami-build-service-eu.herokuapp.com-https
```
Comment (JakeChampion) [inline RUNBOOK.md]: That shouldn't be here, it is for the origami build service 🙈

File: RUNBOOK.md [hunk 3]
```diff
@@ -44,26 +40,20 @@ origami-team
...
+- rowan.manning
 
 ## Dependencies
 
-* origami-navigation-service-data
-* dyn-dns
-* ft-fastly
+- origami-navigation-service-data
+- dyn-dns
+- ft-fastly
```
Comment (JakeChampion) [inline RUNBOOK.md]: Should we add Heroku here?

Comment (JakeChampion) [reply RUNBOOK.md]: And AWS S3?

File: RUNBOOK.md [hunk 4]
```diff
@@ -95,53 +85,51 @@ Manual
...
 See the live service for more information on how to use.
 
 ## First Line Troubleshooting
 
-There are a few things you can try before contacting the Origami team:
-
-1. Verify that GitHub and S3 are up. Either of these being down could cause downtime for this application. See [GitHub's status page](https://www.githubstatus.com/) and the [Navigation Data's `__gtg` endpoint](https://www.ft.com/__origami/service/navigation-data/__gtg).
+There are a few things you can try before contacting the Origami team:
+
+1. Verify that GitHub and S3 are up. Either of these being down could cause downtime for this application. See [GitHub's status page](https://www.githubstatus.com/) and the [Navigation Data's `__gtg` endpoint](https://www.ft.com/__origami/service/navigation-data/__gtg).
```
Comment (JakeChampion) [inline RUNBOOK.md]: I wonder why GitHub being down could cause downtime for Origami-Navigation-Service, I don't think that would be true

Comment (JakeChampion) [reply RUNBOOK.md]: I can't see this codebase making a dependency on Github in production

File: RUNBOOK.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+## More Information
+
+See the live service for more information on how to use.
+
+## First Line Troubleshooting
+
+There are a few things you can try before contacting the Origami team:
+
+1. Verify that GitHub and S3 are up. Either of these being down could cause downtime for this application. See [GitHub's status page](https://www.githubstatus.com/) and the [Navigation Data's `__gtg` endpoint](https://www.ft.com/__origami/service/navigation-data/__gtg).
```
Comment (chee) [inline RUNBOOK.md]:
```suggestion
1. Verify that S3 is up. See [Navigation Data's `__gtg` endpoint](https://www.ft.com/__origami/service/navigation-data/__gtg).
```

**Pull request merged**